flutter/news/pubspec.yaml

```diff
@@ -17,11 +16,10 @@ dependencies:
   get_it: ^7.2.0
   flutter_bloc: ^8.0.1
   dartz: ^0.10.1
-  internet_connection_checker: ^0.0.1+3
   http: ^0.13.4
   shared_preferences: ^2.0.13
   collection: ^1.15.0
-
+  equatable: ^2.0.0
```

Comment:
@Diogo00dev ! I think I failed you on a previous comment about this. I didn't know what equatable was, but I think it's a good idea to use it when we try to define something like a `data class` on flutter. So please, take a look and make sure stuff that we have that are equivalent to a `data class` (for example Articles) uses this stuff

Reply:
i was using equatable in Articles, but had to remove it and replace actually overriding the equals and hashcode, because it was not working properly, so i just decided to remove everything